themes/bootstrap3/templates/Helpers/captcha.phtml

```diff
@@ -0,0 +1,33 @@
+<?php if ($this->wrapHtml):?>
+  <div class="form-group">
+<?php endif;?>
+
+<?php if (count($this->captchas) == 1):?>
+  <label class="control-label"><?=$this->transEsc('captcha_label_single')?></label>
+  <p><?=$this->captcha()->getHtmlForCaptcha($this->captchas[0])?></p>
+<?php else:?>
+  <label class="control-label"><?=$this->transEsc('captcha_label_multiple')?></label>
+  <!-- nav -->
+  <ul class="nav nav-tabs">
+    <?php foreach ($this->captchas as $i => $captcha):?>
+      <?php $active = $i == 0 ? ' class="active"' : ''?>
+      <li<?=$active?>>
+        <a href="#<?=$captcha->getId()?>" role="tab" data-toggle="tab"><?=$captcha->getId()?></a>
```

Comment:
Do you think we should add a translation string for this -- i.e. `<?=$this->transEsc('captcha_method_' . $captcha->getId(), [], $captcha->getId())?>`?

Reply:
(This example will look for a prefixed `captcha_method_` translation string, but default to the raw ID if no prefixed string is found; alternatively, we could also set up a text domain for this, like `<?=$this->transEsc('captcha_method::' . $captcha->getId())?>` ... but adding a whole new .ini file just for these seemed potentially excessive.